packages/design-system/src/components/slider/slider.tsx

```diff
@@ -0,0 +1,312 @@
+import {
```

Comment:
i have a feeling i didn't organize this file properly, please lmk if there is a better way to do this!